### PR TITLE
Enable escrow configuration for different chains (L3 tvl)

### PR DESCRIPTION
**Pull request by antooni**
- [x] double check if ETH on those chains will not be counted as an external/native token in TVL v1

Resolves L2B-4844

### CODE REVIEW

**Review by maciekop-l2b — APPROVED** [review]
LGTM

File: packages/config/src/projects/layer2s/arbitrum.ts [hunk 1]
```diff
@@ -295,12 +295,16 @@ export const arbitrum: Layer2 = {
         batchSize: 150,
         sinceBlock: 7654707,
         version: '3',
+        // TODO: fix this
+        isNativeBalanceSupported: false,
```
Comment (michalsidzej) [inline packages/config/src/projects/layer2s/arbitrum.ts:299]: This is dependant only on the multicall version. V3 does support native balance https://arbiscan.io/address/0xcA11bde05977b3631167028862bE2a173976CA11#code#F1#L199. What's the problem here?

File: packages/backend/src/modules/tvl/modules/ChainTvlModule.ts [hunk 2]
```diff
@@ -108,9 +109,12 @@ export function chainTvlModule(
       logger,
     )
 
-  const circulatingSupplyTokens = tokens.filter(
-    (t) => t.chainId === config.chainId && t.formula === 'circulatingSupply',
-  )
+  const circulatingSupplyTokens = tokens
+    // temporary solution - will be removed once tvl2 migration is complete
```
Comment (michalsidzej) [inline packages/backend/src/modules/tvl/modules/ChainTvlModule.ts:113]: Please create a task for that

File: packages/backend/src/model/Project.ts [hunk 3]
```diff
@@ -117,6 +129,47 @@ function toBackendTrackedTxsConfig(
...
+                .map((tokenSymbol) => {
+                  const token = tokensOnChain.find(
+                    (t) => t.symbol === tokenSymbol,
+                  )
+                  assert(
+                    token,
+                    `${layer3.id}: token with symbol ${tokenSymbol} not found on ${chain}`,
+                  )
+                  return token
+                }),
```
Comment (michalsidzej) [inline packages/backend/src/model/Project.ts:166]: Please make those calculations higher in the tree. It's very hard to see what's going on with this little space in the line left.

File: packages/backend/src/modules/tvl/modules/ChainTvlModule.ts (hunk 2 above)
Comment (antooni) [reply packages/backend/src/modules/tvl/modules/ChainTvlModule.ts:113]: This will be removed once we remove tvl v1

**Review by michalsidzej — APPROVED** [review]

**Pull request merged**